pkgs/applications/editors/vim/plugins/vim-utils.nix

```diff
@@ -415,6 +415,7 @@ rec {
 
   toVimPlugin = drv:
     drv.overrideAttrs(oldAttrs: {
+      name = "vimplugin-${oldAttrs.name}";
```

Comment:
Just accidentally noticed that all treesitter grammars are prefixed with `vimplugin` twice.

https://github.com/NixOS/nixpkgs/blob/7ac1647abc383ad3d6c59cfb57519294d5809143/pkgs/applications/editors/neovim/utils.nix#L221

@teto